pkg/utils/constants/constants.go

```diff
@@ -1,9 +1,12 @@
 package constants
 
 const (
-	AppLabel            = "app"
-	AppLabelValue       = "netobserv-flowcollector"
-	ReporterSource      = "source"
-	ReporterDestination = "destination"
-	ReporterBoth        = "both"
+	AppLabel                  = "app"
+	AppLabelValue             = "netobserv-flowcollector"
+	ReporterSource            = "source"
+	ReporterDestination       = "destination"
+	ReporterBoth              = "both"
+	ApplicationInfrastructure = "infrastructure"
+	ApplicationHosted         = "hosted"
+	ApplicationBoth           = "both"
```

Comment:
hmm I don't know if it's just me lacking of language culture but I don't understand very well the chosen words here (especially "hosted"). Could it be closer to the texts in the UI: scope=infrastructure / scope=applications / scope=both ?
(not super important anyway)

Reply:
I've seen the same wording is used in the frontend .. that's ok I don't want to trigger a full refactoring

Reply:
Well it comes from `hosted application`:
```
A hosted application is any piece of software that is running on another provider’s infrastructure rather than on-premise
```

I admit It can be confusing. Scope is already used for topology and could be in table so I wanted to avoid that generic word here.
If we find a better word I would be happy to rename all :smile:

Reply:
@jpinsonneau You originally had:  
Added `Traffic Scope` query options:

*   Infrastructure
*   Applications
*   Both

It could be "Traffic Type" if you want to avoid the word scope or "Flow Type".  OpenShift Console's terminology is "Applications".

Reply:
Sounds good to me. @jotak is it fine for you ?

Reply:
"Type" is such a versatile word, there's been a couple of times already when I thought about naming something "Flow type" and it was completely different ... so I'm not sure about that. If you ask me "what's a flow/traffic type" without more context, I  would certainly not think about apps/infra scopes

Reply:
sounds good to me!